content/en/synthetics/cicd_testing/ci_results_explorer.md

```diff
@@ -0,0 +1,82 @@
+
+* Drill down into each CI job execution to identify and troubleshoot failing test results
+* Compare test results executed against various devices and browsers to pinpoint cross browser and device issues
+* Keep track of the progress of your tests in CI pipelines
+* Identify flaky tests that might need to be fixed
+
+{{< img src="synthetics/ci/ci_results_explorer/ci_results_explorer.jpeg" alt="CI Results Explorer"  style="width:100%;">}}
+
+## Exploring batches
+
+The CI Results Explorer displays all batches of tests executed via the [Synthetic CI/CD testing integration][1]. Each batch corresponds to a call to the Datadog API (via the NPM package or directly via the API endpoint) to trigger one or several test executions . 
+
+When clicking on a batch, a side panel containing batch CI metadata as well as batch test results shows up, allowing you to look into all test executions that were performed as part of this batch and pinpoint any test failure. You can then click on any failing test result to be redirected to the detailed test result page and look into the root cause of the issue more thoroughly.
+
+## Search
+
+### Facets and tags
+
+The panel on the left side of the page lists several facets you can use to search through your batches:
+
+* **Batch** facets allow you to filter on attributes of your batches:
+
+| Facet            | Description                                                 |
+|------------------|-------------------------------------------------------------|
+| `Summary Status` | The status of the batch: `Passed`, `Failed`, `In Progress`. |
+| `Duration`       | The overall duration of the batch.                          |
+| `ID`             | The id of the batch.                                        |
+
+* **Git** facets allow you to filter on Git related attributes of your batches:
+
+| Facet       | Description                               |
+|-------------|-------------------------------------------|
+| `Branch`    | The branch associated with the batch.     |
+| `Commit SHA`| The commit SHA associated with the batch. |
+
+**CI** facets allow you to filter on CI related attributes of your batches:
+
+| Facet          | Description                                 |
+|----------------|---------------------------------------------|
+| `CI Provider`  | The CI provider associated with the batch.  |
+| `Pipeline URL` | The pipeline URL associated with the batch. |
+
+**Test result** facets allow you to filter on attributes of the test results executed as part of your batches:
+
+| Facet            | Description                                                                                             |
+|------------------|---------------------------------------------------------------------------------------------------------|
+| `Execution Rule` | The execution rule associated with the test result of the batch: `Blocking`, `Non Blocking`, `Skipped`. |
```

Comment:
```suggestion
| `Execution Rule` | The execution rule associated with the test result of the batch: `Blocking`, `Non Blocking`, or `Skipped`. |
```

Reply:
Resolving that one for now as we probably want to keep things consistent with the format we're using on that page https://docs.datadoghq.com/synthetics/search/#facets-and-tags where there's no OR, don't you think?